sources/en/s/scribblehub.py

```diff
@@ -1,22 +1,32 @@
 # -*- coding: utf-8 -*-
+
 import logging
-from math import ceil
-from urllib.parse import quote
+import re
+
+from typing import Generator, Union
+
+from bs4 import BeautifulSoup, Tag
 
-from bs4 import Tag
+from lncrawl.models import Chapter, SearchResult, Volume
+from lncrawl.templates.browser.searchable import SearchableBrowserTemplate
+from lncrawl.core.exeptions import FallbackToBrowser
 
-from lncrawl.core.crawler import Crawler
+from urllib.parse import urljoin, quote_plus
 
 logger = logging.getLogger(__name__)
-chapter_post_url = "https://www.scribblehub.com/wp-admin/admin-ajax.php"
 
+digit_regex = re.compile(r"\?toc=(\d+)#content1$")
 
-class ScribbleHubCrawler(Crawler):
+
+class ScribbleHubCrawler(SearchableBrowserTemplate):
     base_url = [
         "https://www.scribblehub.com/",
         "https://scribblehub.com/",
     ]
```

Comment:
Same thing here, it'd be nice if it could also support `http`.

Reply:
What's the use case for this? An end-user will never paste an `http` scheme URL on a website that has an `https` redirect scheme.

Reply:
let's stay in https